src/merge.js

```diff
@@ -0,0 +1,3 @@
+import merge from 'lodash.merge';
+
+export default merge;
```

Comment:
well merge is a bit more expensive, assign does explicit shallow merge. so I guess it's better.

But for now... Go drunk, you're home!